resources/lang/pl/users.php

```diff
@@ -218,14 +233,24 @@
                 'weighted_pp' => 'ważone: :pp (:percentage)',
             ],
         ],
+        'info' => [
+            'interests' => 'Zainteresowania',
+            'lastfm' => 'Last.fm',
+            'location' => 'Obecna lokalizacja',
+            'occupation' => 'Zajęcia',
+            'skype' => 'Skype',
+            'twitter' => 'Twitter',
+            'website' => 'Strona internetowa',
+        ],
         'page' => [
             'description' => '<strong>ja!</strong> to twoje osobiste, personalizowalne miejsce na twoim profilu.',
```

Comment:
to twoje osobiste miejsce, które możesz dowolnie dostosować.

Ew. początek można zmienić na "Strona <strong>ja!</strong>", jeżeli pasuje, bo nie wiem, gdzie to jest wyświetlane